src/Altinn.Platform/Altinn.Platform.Register/Register/Controllers/PartiesController.cs

```diff
@@ -38,5 +38,39 @@ public async Task<ActionResult> Get(int partyID)
 
             return Ok(result);
         }
+
+        /// <summary>
+        /// Gets the party for a given social security number or organization number.
+        /// </summary>
+        /// <param name="lookupValue">SSN or OrgNumber.</param>
+        /// <returns>The party represeting the provided SNN/OrgNumber.</returns>
+        [HttpGet("lookupObject")]
+        public async Task<ActionResult> LookupPartyBySSNOrOrgNo([FromBody]string lookupValue)
```

Comment:
We should avoid using SSN in description and method names. SSN is specific to US. Instead use PersonNumber. The method could be named: LookupPartyByPersonOrOrganisationNumber